src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -379,6 +396,9 @@ private Object query(Map<String, Object> params) {
     }
 
     Map<String, Serializable> properties = new HashMap<>();
+    if (params.containsKey("properties")) {
```

Comment:
Was this key just not being used before?

Reply:
Yes, it is always passed empty when initializing a new queryRequest to the catalogFramework https://github.com/connexta/ddf-jsonrpc/blob/master/src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java#L384

Reply:
![image](https://user-images.githubusercontent.com/4467636/83084193-25738b80-a03d-11ea-9773-5609b2def9e8.png)
^ Here is an example queryRequest to the catalogFramework not using the rpc

I think we will have to update the getProperties method to also read in other properties as well https://github.com/connexta/ddf-jsonrpc/pull/20/files#diff-4b8f31c7553886d543ac3f68641f3f4eR296